lib/constants.php

```diff
@@ -3,7 +3,7 @@
 namespace Aerys;
 
 const SERVER_NAME = "aerys";
-const SERVER_VERSION = "0.1.0-dev";
+define('SERVER_VERSION', \PackageVersions\Versions::getVersion('amphp/aerys'));
```

Comment:
I think this needs to define the namespace name in the constant, no?

Reply:
Yes, it is right. 
After fix this code get value `0.6.0.9999999-dev@9fbe753c9962eb6b7adc7ae99a6ef917f6b50a80` on my master branch.
Do you like it?